test_runner/src/test/kotlin/ftl/cli/firebase/test/android/versions/AndroidVersionsListCommandTest.kt

```diff
@@ -0,0 +1,29 @@
+package ftl.cli.firebase.test.android.versions
+
+import com.google.common.truth.Truth.assertThat
+import ftl.config.FtlConstants
+import org.junit.Test
+import picocli.CommandLine
+
+class AndroidVersionsListCommandTest {
+
+    @Test
+    fun androidVersionsListCommandOptions() {
```

Comment:
What are the purposes of this test?

Reply:
checked if parameters are set

Reply:
There is no logic under the hood. Why do that?

Reply:
just follow testing practices from other `Command`s
I will remove it

Reply:
Yup, I see. Now I found that we should change those practices. There is no reason for testing assignments.

Reply:
Fixed